source/docs/application-notes/manual-pid-tuning.rst

```diff
@@ -0,0 +1,334 @@
+
+.. raw:: html
+
+    <div class="viz-div" id="vertical_arm_container">
+      <div >
+         <div class="col" id="vertical_arm_plotVals"></div>
+         <div class="col" id="vertical_arm_plotAmps"></div>
+      </div>
+      <div class="flex-grid">
+         <div class="col" id="vertical_arm_viz"></div>
+         <div id="vertical_arm_ctrls"></div>
+      </div>
+      <script>
+         turret = new VerticalArmPIDF("vertical_arm", "both");
+      </script>
+    </div>
+
+.. dropdown:: Tuning Process Example
+
+   Following the guide, I start with all gains at 0 to dial in kG.
+
+   I start with a kG of 1, and notice that the arm's still falling, so I increase it to 2, 4, 8, and 16 before it stops moving. From there I reduce it to 12, then 10 and notice it fall again. I bring it up to 11 and see it still falls appreciably, so I leave it at 12 for the lower bound.
+
+   Going back up, I start at 16 again, then to 18, and 20 before it moves its way up. 19 also produces appreciable movement, so I leave it at 18. This means my kG is (12 + 18) / 2 = 15 amps.
+
+   From here, I set a setpoint of 0.1 and dial in kS to just before it starts moving. I increase it to 1, 2, and 4 when it starts moving. From here, I dial it down to 3 where it doesn't move, and back up to 3.5, 3.7 where it moves again. I check 3.6 and see it doesn't move, so I leave kS at 3.6 amps.
+
+   Now it's time for kP/kD tuning. I bring kP up to 1, 2, 4, 8, 16, and 32 before I get significant overshoot, where I dial kD in to 1, 2, 4, and 8 before that overshoot is gone. kP keeps increasing to 64 and 128, then kD goes up to 16 and 32 before it's back to kP. I go up to 256 and 512 where I notice a bit of oscillation, and I may be near the limit at this point. kD increases to 64 and I get oscillation on the way to the target, so I bring it down to 50 then 40 before I'm happy with it. There's still a little oscillation at the target, but it's minimal.
+
+   I check with other setpoints of -0.1, 0.4, 0.6 and confirm the movement looks good, and say the PID tuning is done.
+
+Profiled Tuning
+^^^^^^^^^^^^^^^
+Profiled tuning can be treated much the same way as tuning a normal PID, but the introduction of a profile means much of the response can be calculated in advance with feed-forwards. This results in most of the work being done due to feed forward, and the feedback gains being used to account for any error in the system.
+
+In Phoenix 6, you can either generate your own profile and feed in the position and velocity setpoints, or use MotionMagic® and let the Talon generate the profile for you. In either case the Talon will have Velocity and/or Acceleration setpoints that it can use the kV and kA feedforward terms on, for more accurate profile following.
+
+The example below uses a pre-generated profile for the system to follow, and the general steps to tune it are below:
```

Comment:
I would recommend stating what each color in the graph represents like was done for the first two examples.

> Red is the position setpoint in rotations, purple is the current position, blue is the velocity setpoint in rotations/sec, brown is the current velocity, yellow is the acceleration setpoint in rotations/sec/sec, orange is the current acceleration, and green is the stator current in amps